opentelekomcloud/services/elb/v3/resource_opentelekomcloud_lb_loadbalancer_v3.go

```diff
@@ -234,9 +240,31 @@ func resourceLoadBalancerV3Read(ctx context.Context, d *schema.ResourceData, met
 
 	publicIpInfo := make([]map[string]interface{}, len(lb.PublicIps))
 	if len(lb.PublicIps) > 0 {
-		info := d.Get("public_ip.0").(map[string]interface{})
-		info["id"] = lb.PublicIps[0].PublicIpID
-		info["address"] = lb.PublicIps[0].PublicIpAddress
+		nwV2Client, err := config.NetworkingV2Client(config.GetRegion(d))
+		if err != nil {
+			return fmterr.Errorf("error creating OpenTelekomCloud NetworkingV2 client: %w", err)
+		}
+		nwV1Client, err := config.NetworkingV1Client(config.GetRegion(d))
+		if err != nil {
+			return fmterr.Errorf("error creating OpenTelekomCloud NetworkingV1 client: %w", err)
+		}
+		floatingIP, err := eips.Get(nwV1Client, lb.PublicIps[0].PublicIpID).Extract()
+		if err != nil {
+			return diag.FromErr(err)
+		}
+		bandwidth, err := bandwidths.Get(nwV2Client, floatingIP.BandwidthID).Extract()
+		if err != nil {
+			return diag.FromErr(err)
+		}
```

Comment:
In this case, it makes sense to use v1 bandwidth to avoid creating second client